jobs/build/tarball-sources/Jenkinsfile

```
@@ -0,0 +1,142 @@
+#!/usr/bin/env groovy
+
+node {
+    checkout scm
+    def release = load("pipeline-scripts/release.groovy")
+    def buildlib = release.buildlib
+    def commonlib = release.commonlib
+
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '')),
+            [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'COMPONENTS',
+                        description: 'Brew component name or NVR',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "logging-fluentd-container"
+                    ],
+                    [
+                        name: 'ADVISORY',
+                        description: 'Image release advisory number. If not given, the number will be retrived from ocp-build-data.',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'OUT_DIR',
+                        description: 'Output dir of elliott tarball-sources',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "/mnt/nfs/home/jenkins/container-sources/"
+                    ],
+                    [
+                        name: 'RCM_GUEST',
+                        description: 'Details of RCM GUEST',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: "ocp-build@rcm-guest.app.eng.bos.redhat.com:/mnt/rcm-guest/puddles/RHAOS/container-sources/"
```

As per [ART-1618](https://issues.redhat.com/browse/ART-1618)
```suggestion
                        defaultValue: "ocp-build@rcm-guest.app.eng.bos.redhat.com:/mnt/redhat/staging-cds/"
```